content/security/encryption-hashing.md

```diff
@@ -59,20 +59,21 @@ Once the installation is complete, you can use the `hash` function, as follows:
 ```typescript
 import * as bcrypt from 'bcrypt';
 
-const saltOrRounds = 10;
+const saltRounds = 10;
+const salt = await bcrypt.genSalt(saltRounds);
 const password = 'random_password';
-const hash = await bcrypt.hash(password, saltOrRounds);
-```
-
-To generate a salt, use the `genSalt` function:
 
-```typescript
-const salt = await bcrypt.genSalt();
+const hash = await bcrypt.hash(password, salt);
+// Store hash in your password DB.
 ```
```

Comment:
This was showing how a salt _could_ be generated if a dev wished to do so. The information was correct